src/parcsr_mv/par_csr_matvec.c

```diff
@@ -12,6 +12,8 @@
  *****************************************************************************/
 
 #include "_hypre_parcsr_mv.h"
+#include "hypre_cuda_utils.h" //RL: TODO par_csr_matvec_device.c, include cuda there
```

Comment:
Looks like you wanted to move this into another file.

Reply:
Yep, there should be a `par_csr_matvec_device.c` which needs the cuda header instead